layouts/partials/head.html

```diff
@@ -19,7 +19,21 @@
   <link rel="stylesheet" href="{{ .Site.BaseURL }}css/poole.css">
   <link rel="stylesheet" href="{{ .Site.BaseURL }}css/syntax.css">
   <link rel="stylesheet" href="{{ .Site.BaseURL }}css/hyde.css">
+  {{ range .Site.Params.custom_css -}}
```

Comment:
What I would suggest is this:

Instead of adding these custom front matter properties that needs to be documented etc., it would be easier and more flexible to do it like this:

https://github.com/bep/docuapi/blob/master/layouts/_default/baseof.html#L10

Just add and include an empty partial that can be overridden by the site.